conductr_cli/conduct_main.py

```diff
@@ -283,7 +283,7 @@ def build_parser(dcos_mode):
 
 def get_cli_parameters(args):
     parameters = ['']
-    if not args.dcos_mode:
+    if not vars(args).get('dcos_mode'):
```

Comment:
dcos_mode is a mandatory field and referenced as such elsewhere. The other fields here are optional.